module/applications/actor/character-sheet.mjs

```diff
@@ -81,7 +81,7 @@ export default class ActorSheet5eCharacter extends ActorSheet5e {
       item.hasUses = uses && (uses.max > 0);
       item.isOnCooldown = recharge && !!recharge.value && (recharge.charged === false);
       item.isDepleted = item.isOnCooldown && (uses.per && (uses.value > 0));
-      item.hasTarget = !!target && !(["none", ""].includes(target.type));
+      item.hasTarget = !!target && !([""].includes(target.type));
```

Comment:
I think this is equivalent?
  - `includes` doesn't need to be used anymore, since we're only checking for one value.
  - `target` should always be an object.
  - `!!target.type` will be false if `target.type` is a blank string, null, undefined, etc.
```suggestion
      item.hasTarget = !!target.type;
```